components/ExampleNav/ExampleNavContainer.coffee

```diff
@@ -4,6 +4,10 @@
 ExampleNav                     = require './ExampleNav'
 
 navs =
+  Dropdowns: [
+    'NavbarDropdownExamples'
```

Comment:
I think NavbarDropdownExamples should be removed from this PR because we are handling it with separate PR and it would create merge conflicts after we merge that other PR.